crates/ruff_python_formatter/src/expression/string.rs

```diff
@@ -467,7 +476,7 @@ fn normalize_string(input: &str, quotes: StringQuotes) -> (Cow<str>, ContainsNew
             newlines = ContainsNewlines::Yes;
         } else if c == '\n' {
             newlines = ContainsNewlines::Yes;
-        } else if !quotes.triple {
+        } else if !quotes.triple && !is_raw {
```

Comment:
Is this the right place for `is_raw`? We can't insert or remove `/` if we're normalizing a raw string.